src/wp-includes/css/wp-tooltip.css

```diff
@@ -0,0 +1,102 @@
+.wp-tooltip-container {
+	position: relative;
+	display: inline-block;
+}
+
+.wp-tooltip-button:focus {
```

Comment:
This should inherit existing styles for buttons, so that :focus states are standardized. Probably add the class 'button' to the trigger and only add additional styles if there is something that needs to be overwritten.